### DISCUSSION TITLE
对于遇到烂尾房的业主深表同情

### DISCUSSION THREAD

**Discussion by zning1994** [WeNeedHome/SummaryOfLoanSuspension#96] [Ideas]
RT

**Comment by dukelec** [comment]
小粉紅除外

**Reply by wengxianxun** [reply]
繁体字，蚌埠住了

**Comment by AntEffect** [comment]
> 小粉紅除外

小粉红:别发外网

**Reply by waxeh0043** [reply]
发，多发，像大翻译运动一样使劲发。
反正到外国人越讨厌中国，高华挨的打越狠，黑人打的是高华，白人做肥皂的也是高华。
不过国外也不允许你发的，因为你觉得的负面新闻在推眼里是为中国政府做宣传，然后大翻译运动被封号了。

**Reply by AntEffect** [reply]
比起高华，更可恨的不应该是红x代和各路官僚吗？

**Reply by waxeh0043** [reply]
但是你的做法，发外网是没一点用处的，他们就是知道了也不会关注也不会宣传，因为他们现在的宣传核心就是攻击一个不存在的目标，比如新疆种族灭绝，但是绝对不会真的宣传比如空气污染一样的问题，因为现实存在的问题他们提出来了，我们这边的高层真的会解决问题。

所以指望洋大人来主持公道，这种想法十分的可笑！

**Reply by AntEffect** [reply]
> 但是你的做法，发外网是没一点用处的，他们就是知道了也不会关注也不会宣传，因为他们现在的宣传核心就是攻击一个不存在的目标，比如新疆种族灭绝，但是绝对不会真的宣传比如空气污染一样的问题，因为现实存在的问题他们提出来了，我们这边的高层真的会解决问题。
> 
> 所以指望洋大人来主持公道，这种想法十分的可笑！

哈哈，那为啥国内会有洋人代报案的现象呢？

**Reply by waxeh0043** [reply]
前面已经说了：“因为现实存在的问题他们提出来了，我们这边的高层真的会解决问题。”
问题是他们已经不愿意提出真正的问题了，只愿意虚空打打靶。

**Reply by trialley** [reply]
> 但是你的做法，发外网是没一点用处的，他们就是知道了也不会关注也不会宣传，因为他们现在的宣传核心就是攻击一个不存在的目标，比如新疆种族灭绝，但是绝对不会真的宣传比如空气污染一样的问题，因为现实存在的问题他们提出来了，我们这边的高层真的会解决问题。
> 
> 所以指望洋大人来主持公道，这种想法十分的可笑！

是啊，真正的问题，外国媒体也不会去管，可精明着。要解决问题只有靠老百姓自己争取。
